gulp.js

```diff
@@ -0,0 +1,68 @@
+var gulp = require('gulp');
+var livereload = require('gulp-livereload');
+var jshint = require('gulp-jshint');
+var stylish = require('jshint-stylish');
+
+/*
+PASTE <script src="//localhost:35729/livereload.js"></script>
+IN THE HEAD OF THE PAGE FOR LIVERELOAD TO WORK
+ */
+
+
+var paths = {
+  images: 'client/public/assets/img/*.*',
+  styles: 'client/public/assets/css/*.css',
+  reactScripts: 'client/public/app/**/*.js',
```

Comment:
I don't recall talking about an app folder within public, but we can use it if you like. Let me know so that I can structure the folders for my work too.